Firestore/Example/Tests/Util/FSTIntegrationTestCase.mm

```diff
@@ -325,6 +325,18 @@ - (void)waitUntil:(BOOL (^)())predicate {
   return result;
 }
 
+extern "C" NSArray<NSArray<id> *> *FIRQuerySnapshotGetDocChangesData(FIRQuerySnapshot *docs) {
+  NSMutableArray<NSMutableArray<id> *> *result = [NSMutableArray array];
+  for (FIRDocumentChange *docChange in docs.documentChanges) {
+    NSMutableArray<id> *docChangeData = [NSMutableArray array];
```

Comment:
Two bits of feedback here:

1. You can create an array of known contents with far less typing by using an array literal, like so:

```
NSArray<id> *docChangeData = @[
  @(docChange.type), docChange.document.documentID, docChange.document.data
];
```

2. I'm not wild about making this untyped and unstructured. Historically the way we've done this is to make a test helper for concisely creating the expected result, usually something like what's in FSTHelpers.h and then defining isEqual/describe on it such that we can XCTAssertEqualObjects and have the right thing happen.

Reply:
Done.